src/routes/api/tasks/index.ts

```diff
@@ -360,4 +343,30 @@ function isErrnoException (error: unknown): error is NodeJS.ErrnoException {
   return error instanceof Error && 'code' in error
 }
 
+function buildFilePath (filename: string, fastify: FastifyInstance) {
+  return path.join(
+    import.meta.dirname,
+    '../../../..',
+    fastify.config.UPLOAD_DIRNAME,
+    fastify.config.UPLOAD_TASKS_DIRNAME,
+    filename
+  )
+}
+
+async function deleteFile (fileName: string, fastify: FastifyInstance) {
+  const filePath = buildFilePath(fileName, fastify)
+
+  try {
+    await fs.promises.unlink(filePath)
+  } catch (err) {
+    if (isErrnoException(err) && err.code === 'ENOENT') {
+      // A file could have been deleted by an external actor, e.g. system administrator.
+      // We log the error to keep a record of the failure, but consider that the operation was successful.
+      fastify.log.warn(`File path '${fileName}' not found`)
+    } else {
+      throw err
+    }
+  }
+}
```

Comment:
I think we should create a new custom plugin for file operations .

We could use it a similar way:
```ts
fastify.fileHandler.uploadTaskImage(filename)
fastify.fileHandler.removeTaskImage(filename)

// Later
fastify.fileHandler.uploadUserImage(filename)
fastify.fileHandler.removeUserImage(filename)
```

Or:
```ts
fastify.fileHandler.upload(taskDirName, filename)
fastify.fileHandler.remove(taskDirName, filename)
```